identity/app/controllers/EditProfileController.scala

```diff
@@ -71,10 +90,10 @@ class EditProfileController(
           ProfileForms(userDO, activePage = page).bindFromRequestWithAddressErrorHack(request) // NOTE: only active form is bound to request data
 
         boundProfileForms.activeForm.fold(
-          formWithErrors => Future(profileFormsView(page, boundProfileForms, userDO)),
+          formWithErrors => profileFormsView(page, boundProfileForms, userDO),
           success = {
             case formData: AccountFormData if formData.deleteTelephone =>
-              identityApiClient.deleteTelephone(userDO.auth) map {
+              identityApiClient.deleteTelephone(userDO.auth) flatMap {
```

Comment:
why the change from map -> flatMap?

Reply:
`profileFormsView` now returns `Future` se we get `Future[Future[_]]`